pkg/controllers/awsloadbalancercontroller/rbac_rules.go

```diff
@@ -35,11 +35,6 @@ func getControllerRules() []rbacv1.PolicyRule {
 			Resources: []string{"pods/status"},
 			Verbs:     []string{"patch", "update"},
 		},
-		{
-			APIGroups: []string{""},
-			Resources: []string{"secrets"},
-			Verbs:     []string{"get", "list", "watch"},
-		},
```

Comment:
Why the operand doesn't need this permissions anymore?

Reply:
This is somewhat unrelated. But the upstream dropped those permissions and the operand was still trying to create the cluster role with that permission which caused it to fail.